src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1312,7 +1312,8 @@ IceUdpTransportManager getTransportManager(
                 try
                 {
                     transportManager
-                        = new IceUdpTransportManager(this, initiator, 1);
+                        = new IceUdpTransportManager(
+                            this, initiator, 1, channelBundleId);
```

Comment:
The TransportManager needs to advertise the URL which endpoints can connect to. The URL encodes the conference and endpoint ID. And channelBundleId coincides with the endpoint ID.